src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -10,6 +10,7 @@ const INITIAL_FACILITY_DISPLAY_COUNT = 5;
  * form system.
  */
 export default function FacilitiesRadioWidget({
+  id,
```

Comment:
Also, I think your current unit tests are failing because of the `useEffect` call. The `scrollAndFocus` call is using the old id style.

Reply:
Done.